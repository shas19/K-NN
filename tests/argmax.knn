let t = initArray(4) in 
let t[0] = 4.0 in
let t[1] = 3.0 in 
let t[2] = 7.0 in
let t[3] = 6.0 in 
ArgMax(t)