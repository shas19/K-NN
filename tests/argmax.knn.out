<T>
  <k>
    2 ~> .
  </k>
  <env>
    t |-> 0
  </env>
  <store>
    0 |-> tensor ( 1 , 4 )
    1 |-> 0.40000000000000000e1
    2 |-> 0.30000000000000000e1
    3 |-> 0.70000000000000000e1
    4 |-> 0.60000000000000000e1
  </store>
  <nextLoc>
    5
  </nextLoc>
  <shape>
    t |-> 4
  </shape>
</T>
